webrtc.html

```diff
@@ -1862,6 +1900,15 @@ <h4>Set the RTCSessionDescription</h4>
                         <a href="#transceivers-set">set of transceivers</a> run the
                         following steps:</p>
                         <ol>
+                          <li>
+                            <p>Set <var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\StableStateSenderTransport]]</a>
+                            to <code>null</code>.
```

Comment:
Resetting these internal slots is not strictly necessary: When we are in the stable state it doesn't matter what they are since they are only read from when we are performing rollback, at which point they have already been set (when applying the offer).

But conceptually these slots are not used unless we are performing an O/A exchange so I thought I would clean it up in all code paths where we return to stable state.

If you think this is unnecessary let me know and I'll remove these steps. (They don't have to be nulled above either.)

Reply:
Sorry for the delay in looking at these. I see this has already merged, but decided to answer here. Maybe we can clean this up in https://github.com/w3c/webrtc-pc/pull/2384.

I ran across this new language while reviewing https://github.com/w3c/webrtc-pc/pull/2384 and admit I got a bit confused by it. We have so many different states (current/pending), and at first I didn't get the narrow scope for these new ones, and found it hard to answer e.g. why we're still returning *SenderTransport* and not *StableStateSenderTransport* in getters, but then I realized these are *backup* slots, which helps understand their narrow scope.

I recommend the following:
1. Since these are backup slots, s/`StableStateSenderTransport`/`LastStableSenderTransport`/
2. Since the algorithm is getting long, skip setting them to `null`.

Reply:
Thanks. To be addressed in #2384.